src/liquibase.ts

```diff
@@ -83,8 +83,8 @@ export class Liquibase {
    *
    * {@link https://docs.liquibase.com/commands/community/update.html Documentation}
    */
-  public update(params: UpdateCommandAttributes): Promise<string> {
-    return this.run(LiquibaseCommands.Update, params);
+  public update(params: UpdateCommandAttributes, substitutionParams?: { [key: string]: any }): Promise<string> {
```

Comment:
@ijakab @vuki656 I'm wondering if there is another way of approaching this, rather than adding an optional param to (i'm assuming) every public method 🤔 .

I have some free time this afternoon that I'll block off to work on this.

Reply:
Did you manage to find another way to approach to this problem ?